scripts/package.sh

```diff
@@ -29,13 +29,14 @@ if [ "$BUILD_TYPE" != "tar" ]; then
         --vendor "Mozilla" \
         --url "https://wiki.mozilla.org/Socorro" \
         --description "$DESC" \
-        --before-install scripts/package/before-install.sh \
-        --after-install scripts/package/after-install.sh \
-        --before-remove scripts/package/before-remove.sh \
-        --after-remove scripts/package/after-remove.sh \
+        --before-install scripts/package/$BUILD_TYPE-before-install.sh \
+        --after-install scripts/package/$BUILD_TYPE-after-install.sh \
+        --before-remove scripts/package/$BUILD_TYPE-before-remove.sh \
+        --after-remove scripts/package/$BUILD_TYPE-after-remove.sh \
         --config-files /etc/socorro \
         --exclude *.pyc \
         --exclude *.swp \
+	--deb-field 'Depends: libc6 (>= 2.4), libgcc1 (>= 1:4.1.1), libpq5 (>= 8.3~), libsasl2-2 (>= 2.1.24), libssl1.0.0 (>= 1.0.0), libstdc++6 (>= 4.5), libxml2 (>= 2.7.4), libxslt1.1 (>= 1.1.26), zlib1g (>= 1:1.2.0), openjdk-7-jre-headless, python-virtualenv, postgresql-9.3, postgresql-plperl-9.3, postgresql-contrib-9.3, rsync, rabbitmq-server, elasticsearch, memcached, apache2, libapache2-mod-wsgi, nodejs, node-less' \
```

Comment:
Thanks for tracking down all these dependencies! (we should really do the same for the RPM)